mitosheet/src/mito/components/taskpanes/UserDefinedImport/UserDefinedImportTaskpane.tsx

```diff
@@ -71,7 +71,13 @@ const getParamTypeDisplay = (
     }
 }
 
-const NO_IMPORTERS_MESSAGE = 'You have not defined any importers. To define importers, pass them to the mitosheet.sheet call with the `importers` parameter. An importer is just a function that returns a pandas dataframe.';
+let NO_IMPORTERS_MESSAGE = 'You have not defined any importers. An importer is just a function that returns a pandas dataframe.';
+if (isInStreamlit()) {
+    NO_IMPORTERS_MESSAGE += ' You can define importers in the mito_component call with the `importers` parameter.';
```

Comment:
We could add a similar check / message in the update files taskpane where we say

<img width="326" alt="Screenshot 2023-07-18 at 11 33 06 AM" src="https://github.com/mito-ds/monorepo/assets/18709905/64abc31a-be22-4d36-abb2-8209b0c489d8">